.cirrus.yml

```diff
@@ -21,9 +21,10 @@ task:
   modules_cache:
     fingerprint_script: cat go.sum
     folder: $GOPATH/pkg/mod
-  get_script: go get ./...
-  build_script: go build ./...
-  test_script: go test ./...
+  test_script:
+    - apt-get update
+    - apt-get install -y zsh
```

Comment:
should we just test it on FreeBSD like in #99? Anyways the issue was raised by a FreeBSD user

Reply:
Why not test it on Linux too? It's almost free.

Reply:
Also a better test case wouldn't require Zsh at all, but I don't have any ATM.